docs/guides/preinstall.md

```diff
@@ -13,68 +23,109 @@ In macOS Disk Utility, [create a partition](https://support.apple.com/guide/disk
 
-Follow the instructions in this section to be followed on macOS [here](https://wiki.t2linux.org/guides/wifi-bluetooth/#on-macos). The sections of that page to be followed on Linux can be followed later, once you've successfully installed Linux.
+The **second part "On Linux"** must be followed **after** you have completed the Linux installation.
 
 ## Disable Secure Boot
 
-!!! note
-    Wait until you have finished creating your install USB and you have copied Wi-Fi firmware before proceeding to this section.
+Now that you have completed the first part of the Wi-Fi firmware in macOS, you can proceed with the disabling of the secure boot.
+Apple's Secure Boot implementation does not allow booting anything other than macOS or Windows when it is enabled (not even shim signed GRUB).
+We need to disable it:
 
-Apple's Secure Boot implementation does not allow booting anything other than macOS and Windows when Secure Boot is enabled (not even shim signed GRUB).
+1. Turn off your Mac
+2. Turn it on and rapidly press and hold `Command-R` until the black screen flashes
+3. Your Mac will boot in the macOS Recovery
+4. Select your user and enter your password
+5. Now, from the menu bar choose Utilities > Startup Security Utility
+6. Enter again the password
+7. Once in Startup Security Utility:
 
-1. Follow [this article's](https://support.apple.com/HT208198) instructions to boot into macOS Recovery and open Startup Security Utility.
-2. Once in Startup Security Utility, set Secure Boot to "No Security" and select "Allow booting from external or removable media". This will allow you to boot from a Linux install ISO.
+   - set Secure Boot to **No Security**
+   - set Allow Boot  Media to **Allow booting from external or removable media**
+
+Now you are able to boot from a Linux install ISO.
 
 !!! Note "Keeping your Mac secure while Secure Boot is off"
-    If you are worried about the reduced security, in Startup Security Utility you can select "Turn On Firmware Password" to require entering a password to boot anything other than the default OS (Currently that would be macOS). After installing Linux on the internal SSD, you have the option to reselect "Disallow booting from external or removable media" in Startup Security Utility.
+    If you are worried about the reduced security, in Startup Security Utility you can select "Turn On Firmware Password" to require entering a password to boot anything other than the
+    default OS. Additionally, after installing Linux on the internal SSD, you will have the option to reselect "Disallow booting from external or removable media" in the Startup Security
+    Utility.
 
 ## Booting your Linux install USB
 
 1. Ensure the Linux Installation USB you created is plugged into your Mac.
 2. Reboot while holding down the option (⌥) key, this will put you in macOS Startup Manager.
-3. Select the orange EFI option with arrow keys and press return/enter on it. If there are two, try the one to the very right first.
```

Comment:
Why removed try? Second one the generally the one, but in some cases the first one also works.

Reply:
I think you are right, added try! [345547e](https://github.com/gianmaticus/wiki/commit/345547e92f485fc37c97359106ce0aa72ef19b95)